src/nbl/CMakeLists.txt

```diff
@@ -575,6 +575,125 @@ if(_NBL_COMPILE_WITH_GLI_)
 	target_include_directories(Nabla PUBLIC ${THIRD_PARTY_SOURCE_DIR}/gli)
 endif()
 
+# DXC
+if(_NBL_COMPILE_WITH_DXC_)
+	add_dependencies(Nabla dxc)
+	target_include_directories(Nabla PRIVATE 
+		"${DXC_BINARY_DIR_GE}/tools/clang/tools/dxcompiler"
+		"${DXC_SOURCE_DIR}/tools/clang/tools/dxcompiler"
+		"${DXC_SOURCE_DIR}/tools/clang/include"
+		"${DXC_BINARY_DIR_GE}/tools/clang/include"
+		"${DXC_SOURCE_DIR}/include/llvm/llvm_assert"
+		"${DXC_BINARY_DIR_GE}/include"
+		"${DXC_SOURCE_DIR}/include"
+		"${DXC_BINARY_DIR_GE}/include/dxc/Tracing"
+		"${DXC_BINARY_DIR_GE}/utils/version"
+	)
+	target_link_libraries(Nabla INTERFACE "${DXC_DXCOMPILER_LIB}")
+	
+	############################################################################
+	# list files you want to compile with DXC in _NBL_ENABLE_DXC_COMPILE_TESTS_
+	# once we have guarantee all of shaders in Nabla can be compiled with DXC,
+	# we can simply grep all of those files/use existing vars holding paths to
+	# shaders and remove manual adding the files bellow
```

Comment:
no because certain shaders will have to be compiled with a different target (`-T` option) and with a actual entry point + spirv gen

Reply:
resolved